wrapper/src/libvmaf.h

```diff
@@ -99,23 +133,24 @@ class Result
     double get_score(const std::string &key);
     std::vector<std::string> get_keys();
     void setScoreAggregateMethod(ScoreAggregateMethod scoreAggregateMethod);
+    int get_num_frms();
+    void set_num_frms(int num_frms);
```

Comment:
Not added by your changes, but I've noticed that these cpp classes are internal only. Please remove them from the public header.

Reply:
Moved a number of stuff into vmaf.h, can improve on the cleanup

Reply:
Cleanup of the rest of the repo todo.